explainaboard/processors/processor.py

```diff
@@ -35,11 +37,19 @@ def __init__(self) -> None:
         self._tokenizer = SingleSpaceTokenizer()
         self._user_defined_feature_config = None
 
-    def _init_statistics(
-        self, sys_info: SysOutputInfo, statistics_func: Callable
-    ) -> Optional[Dict]:
-        """Take in information about the system outputs and a statistic calculating function and return a dictionary
-        of statistics.
+    def _get_statistics_resources(
+        self, dataset_split: Dataset
+    ) -> Optional[Mapping[str, Any]]:
+        """
+        From a DataLab dataset split, get resources necessary to calculate statistics
+        """
+        return None
+
+    def _gen_external_stats(
```

Comment:
there is a `statistics` inside this function,  would it be a good idea to rename them as `external_stats` in the future?